code/src/index.css

```diff
@@ -1,13 +1,419 @@
+  }
+
+
+@keyframes gradient {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+
+}
+}
+
+/*TABLET*/
+@media (min-width: 650px) {
+
+  .main-container {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .header-container {
+    width: 100%;
+  }
+
+  .header-title {
+    width: 100%;
+    background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+    background-size:  400% 400%;
+  animation: gradient 15s ease infinite;
+  background-clip: text;
+  -webkit-background-clip: text;
+  -moz-background-clip: text;
+  -webkit-text-fill-color: transparent; 
+  -moz-text-fill-color: transparent;
+    font-size: 70px;
+    font-family: 'Sora', sans-serif;
+  }
+
+
+  @keyframes gradient {
+    0% {
+      background-position: 0% 50%;
+    }
+    50% {
+      background-position: 100% 50%;
+    }
+    100% {
+      background-position: 0% 50%;
+    }
+  }
+
+
+  .header-line {
+    width: 86%;
+    margin: 0;
+  }
+
+  .album-card {
+    width: 40%;
+  }
+
+  .icons-container {
+    top: 45%;
+  }
+
+  .icons-container img {
+    width: 50px;
+    margin: 0 10px;
+  }
+}
+
+
+
+/*DESKTOP*/
+@media (min-width: 992px) {
+
+  .album-card {
+    width: 26%;
+  }
+
+  .icons-container {
+    top: 45%;
+  }
+
+  .icons-container img {
+    width: 40px;
+    margin: 0 10px;
+  }
+
+  .header-title {
+    width: 100%;
+    background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+    background-size:  400% 400%;
+    animation: gradient 15s ease infinite;
+    background-clip: text;
+    -webkit-background-clip: text;
+    -moz-background-clip: text;
+    -webkit-text-fill-color: transparent; 
+    -moz-text-fill-color: transparent;
+    font-size: 90px;
+    margin-top: 30px;
+    font-family: 'Sora', sans-serif;
+  }
+
+
+
+@keyframes gradient {
+  0% {
+    background-position: 0% 50%;
+  }
+  50% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+}
+}
+
+
+/*DESKTOP BIG SCREEN*/
+@media (min-width: 1400px) {
+
+  .album-card {
+    width: 20%;
+  }
+
+  .icons-container {
+    top: 45%;
+  }
+
+  .icons-container img {
+    width: 40px;
+    margin: 0 10px;
+  }
+
+  .header-title {
+    width: 100%;
+    background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+    background-size:  400% 400%;
+    animation: gradient 15s ease infinite;
+    background-clip: text;
+    -webkit-background-clip: text;
+    -moz-background-clip: text;
+    -webkit-text-fill-color: transparent; 
+    -moz-text-fill-color: transparent;
+    font-size: 120px;
+    margin-top: 30px;
+    font-family: 'Sora', sans-serif;
+  }
+
+  @keyframes gradient {
+    0% {
+      background-position: 0% 50%;
+    }
+    50% {
+      background-position: 100% 50%;
+    }
+    100% {
+      background-position: 0% 50%;
+    }
+  }
```

Comment:
The only criticism I have regarding the CSS is I would maybe break it up into smaller pieces and have each and every component have its own corresponding CSS-file. I've found that that really makes it easier (for me at least) to find what I'm looking for when styling, and prevents endless scrolling. Other than that the site looks great as usual and the React code is smooth and well done.